src/app/components/letters/create-letters-temp/create-letters-temp.component.ts

```diff
@@ -1,12 +1,53 @@
 import { Component } from '@angular/core';
+import { statuses } from 'src/app/global/constants' ;
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { FormBuilder } from '@angular/forms';
+import { Router } from '@angular/router';
+
 
 @Component({
   selector: 'app-create-letters-temp',
   templateUrl: './create-letters-temp.component.html',
   styleUrls: ['./create-letters-temp.component.scss']
 })
-export class CreateLettersTempComponent  {
+export class CreateLettersTempComponent {
+
+
+  constructor(private fb: FormBuilder,
+              private router: Router) {}
+
+   profileForm = this.fb.group( {
+     name: ["",[
```

Comment:
Please use double quotes in html, and single quotes in typescript files

Reply:
done